### DISCUSSION TITLE
Examples of BlendMode in NFTChef

### DISCUSSION THREAD

**Discussion by nerdylist** [nftchef/art-engine#37] [Help + Q&A] [answered]
Hello, could you please give an example of the right way to do BlendMode in NFTChef?

Thanks!

**Comment by nftchef** [comment, marked as answer]
Hi, blend modes can be passed in at the layerOrder object, either to an entire `layer` using the blend property,

```js
   layersOrder: [
      { name: "Background" },
      { name: "Hats", blend: 'multiply' },
      { name: "Male Hair" },
]
 ```

Or to subfolders using the sublayer options

```js
   layersOrder: [
      { name: "Background" },
      { name: "Hats", sublayerOptions: { SubFolderName: { blend: "source-over" } } },
      { name: "Male Hair" },
]
```

**Reply by nerdylist** [reply]
Would it then be       { name: "Hats", blend: 'multiply', opacity: '.5' },  ... If I wanted to change opacity along with blend mode?

**Reply by nftchef** [reply]
that's correct
